test/test_selectors.py

```diff
@@ -613,6 +612,34 @@ def test_leaking_fds(self):
         after_fds = len(proc.open_files())
         self.assertEqual(before_fds, after_fds)
 
+    @skipIf(sys.platform == "win32", "psutil.Process.open_files() is unstable on Windows.")
+    def test_wait_read_leaking_fds(self):
+        old_selector = wait.DefaultSelector
+        wait.DefaultSelector = self.SELECTOR
```

Comment:
Otherwise each selector type won't be tested, only whatever the default is times the number of available selectors.